### ISSUE TITLE
How to populate mandatory MedicationRequest.intent

### ISSUE THREAD

**Issue by ArdonToonstra** [Nictiz/Nictiz-R4-zib2020#127]
MedicationRequest has the mandatory element `intent` describing if the request is a proposal, plan, or an original order. Amongst others, it contains the following relevant codes in this [valueset](http://hl7.org/fhir/valueset-medicationrequest-intent.html):

Code | Display | Definition
-- | -- | --
proposal | Proposal | The request is a suggestion made by someone/something that doesn't have an intention to ensure it occurs and without providing an authorization to act
plan | Plan | The request represents an intention to ensure something occurs without providing an authorization for others to act.
order | Order | The request represents a request/demand and authorization for action
original-order | Original Order | The request represents the original authorization for the medication request.

'* original-order is new as of R4

The zibs MedicationAgreement and DispenseRequest are mapped to this resource. Can we provide our implementers guidance on which code they should use with each zib?

Our thoughts are:
* MedicationAgreement would have an intent = _plan_ (or _order_ /_original-order_ ? )
* DispenseRequest would have an intent = order / _original-order_

**Comment by ArdonToonstra** [comment]
@MarijkeVanGeijn @weteringac  would you mind thinking along with providing the best guidance on the use of MedicaitonRequest.intent?

**Comment by weteringac** [comment]
I believe both MA and VV authorise action (for patient/pharmacist/professional administrator/et cetera), so both should map on order.

**Comment by ArdonToonstra** [comment]
Thanks. It is also the code that is fixed in STU3. 
I have removed the fixed code and placed guidance on the comment element on how to fill the `intent`, like this:

>Unless `intent` is explicitly recorded and a more appropriate code is known, the value can be set to _order_ because a MedicationAgreement should authorize an action for a patient, pharmacist, professional administrator et cetera.

I removed the fixed code to align more with the open world modelling principle. If systems (in the future) have implemented workflow and contain more appropriate codes, they can reuse this profile for that. 

I also adjusted this in the mappings. Will close this issue.
